velodyne_pcl/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>velodyne_pcl</name>
+  <version>0.0.0</version>
```

Comment:
Please set to version 1.5.2 to match existing version for ease of upgrade with `catkin` tools.